packages/commons/src/auth/jwk.ts

```diff
@@ -64,3 +66,17 @@ export function sortJWK(jwk: JsonWebKey): JsonWebKey {
       {}
     );
 }
+
+export function getJwksClient(): JwksClient[] {
+  const config = JWTConfig.parse(process.env);
+  return config.wellKnownUrls.map((url) =>
+    jwksClient({
+      cache: true,
+      cacheMaxEntries: 50,
+      timeout: 30000,
```

Comment:
We could not set these and rely on the default values here unless we have a reason to set different values